packages/esm-appointments-app/src/form/appointments-form.resource.ts

```diff
@@ -1,4 +1,6 @@
+import { useCallback } from 'react';
 import dayjs from 'dayjs';
+import isToday from 'dayjs/plugin/isToday';
```

Comment:
Not germane to this PR, but we install a bunch of dayjs plugins in core. We should probably figure out how to properly export those types so it's not necessary to do in each esm.